vms/platformvm/vm_test.go

```diff
@@ -2084,7 +2085,7 @@ func TestUptimeDisallowedAfterNeverConnecting(t *testing.T) {
 	require.NoError(abort.Accept(context.Background()))
 	require.NoError(vm.SetPreference(context.Background(), vm.manager.LastAccepted()))
 
-	// Verify that rewarded validator has been removed.
+	// verify that rewarded validator has been removed.
```

Comment:
So Stephen's comment made me revert the export of the `verify` function, and the IDE decided to change all `verify` to `Verify` it could find. 

I thought I reverted its annoying mistake properly, but seems like I missed some. Fixed, thanks.